webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js

```diff
@@ -12,6 +12,9 @@ describe('medic-xpath-extensions', function () {
   };
 
   describe('getTimezoneOffsetAsTime()', function () {
+    const getTimeZoneOffset = Date.prototype.getTimezoneOffset;
+    afterEach(() => Date.prototype.getTimezoneOffset = getTimeZoneOffset);
```

Comment:
This was the cause of _some_ of the weirdness below (and I think fixing it also maybe caused some of the additional failures I uncovered),  Either way, I figure we def want to be re-setting `getTimezoneOffset` here.